osmaxx/utils/frozendict.py

```diff
@@ -0,0 +1,12 @@
+from types import MappingProxyType as dictproxy  # noqa
```

Comment:
> is the noqa for lower_casing a ClassName?
Yes

> And why are we lowercasing this in the first place?
* So the code I copy-pasted below that can work unmodified.
* So we could switch to dictproxyhack by only changing the import line (and `requirements.txt`)
* So it is consistent with `dict()`, `list()` etc.